handbook/company.md

```diff
@@ -78,27 +78,27 @@ Empathy leads to better understanding, better communication, and better decision
   - Own up to mistakes.
   - Understand why it matters (the goals of the work you are doing).
   - Consider the business impact (fast forward 12 months, consider the total cost of ownership over the eternity of maintenance).
-  - Do things that don't scale, sometimes.
+  - Do things that don't scale, sometimes. 
 - Be responsive.
   - Respond quickly, even if you can't take further action at that exact moment.
   - When you disagree, give your feedback; then agree and commit, or disagree and commit anyway.
   - Favor short calls over long asynchronous back and forth discussions in Slack.
   - Procrastination is a symptom of not knowing what to do next (if you find yourself avoiding reading or responding to a message, schedule a Zoom call with the people you need to figure it out).
 - We win or lose together.
-  - Think about the big picture, beyond your individual team's goals.
+  - Think about the big picture beyond your individual team's goals.
   - Success equals creating value for customers.
-  - You're not alone in this (there's a great community of people able and happy to help).
+  - You're not alone in this (a great community of people who are able and happy to help).
   - Don't be afraid to spend time helping users, customers, and contributors (including colleagues on other teams).
   - Be proactive (ask other contributors how you can help, regardless of who is assigned to what
-  - Finish tasks all the way before moving to something new (help unblock team members and other contributors to deliver value). 
+  - Finish completely way before moving to something new (help unblock team members and other contributors to deliver value). 
```

Comment:
You don't have to finish completely _way_ before moving on, you just have to finish completely, then move on.